src/psyclone/gocean1p0.py

```diff
@@ -892,8 +896,25 @@ def reference_accesses(self, var_accesses):
         :type var_accesses: \
             :py:class:`psyclone.core.access_info.VariablesAccessInfo`
         '''
+
+        # Grid properties are accessed using one of the fields. This stores
+        # the field used to avoid repeatedly determining the best field:
+        grid_for_property = None
```

Comment:
Strictly speaking, this should be `field_for_grid_property` or somesuch.

Reply:
Indeed ... though it made it necessary to split a line ... 81 characters with the better name